h/static/scripts/tests/util/submit-form-test.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var submitForm = require('../../util/submit-form');
+
+var lastRequest;
+function FakeXMLHttpRequest() {
```

Comment:
You are aware that sinon [has it's own fake XHR module](http://sinonjs.org/docs/#server)? We don't have to write everything ourself.